meson.build

```diff
@@ -1,5 +1,5 @@
 project('pipewire', ['c' ],
-  version : '0.2.9',
```

Comment:
I'm a bit confused. Have there been 90 releases of 0.2.x, or why do you want to set it to `90`? From the PR description I'm a bit unclear what are you trying to solve.

Reply:
I also think it would be useful to have explanation of the commit in commit message too.

Reply:
The .90 is usually used for pre-releases so 0.2.90 is a pre-release of 0.3. If one wants to use ifdefs to support both 0.2 and 0.3, the version fo the first 0.3 release (even for pre-release) has to be increased. I cannot check for 0.3 as this was not released yet and there are already releases with different API.

Reply:
> The .90 is usually used for pre-releases so 0.2.90 is a pre-release of 0.3

Okay, I might say something silly since I couldn't find what release scheme Pipewire uses. But for example Linux kernel increments the last number for every bugfix release, so `.90` would mean 90 bugfix releases. Same for Mesa, except they don't support older releases long enough to reach that big number.

The only project I know that uses the scheme you mention is GTK, and I find it utterly confusing. Because you think "okay, I gonna take latest GTK3", and then you find out that the "latest GTK3 release" is actually a pre-release to GTK4, so everything gets broken.

Reply:
Not only GTK, we use it for example in KDE.

And given 0.2 branch/version is dead by now, I don't expect 83 more bugfix releases. I opened a PR on gitlab, let's see what Wim has to say.